tool/gravity/cli/config.go

```diff
@@ -157,9 +158,22 @@ type InstallConfig struct {
 	FromService bool
 	// AcceptEULA allows to auto-accept end-user license agreement.
 	AcceptEULA bool
+
+	// Computed values
+	//
 	// writeStateDir is the directory where installer stores state for the duration
 	// of the operation
 	writeStateDir string
+	// app is the application being installed
+	app *app.Application
```

Comment:
I would make this a value instead of a pointer, to avoid potential panics if the caller doesn't check this for non-nil.

Reply:
The point of handling this is pretty limited so a somewhat skilled coder can manage to avoid the nil pointer. It is also exacerbated by the fact that `lib/install.Config.App` is a pointer attribute to validate that it's been properly set - not sure which pattern is better here - using a value and some way to tell an empty struct from a non-empty one or leaving a pointer...

Reply:
Ok, sounds good.